ansible/roles/nginx/templates/site.j2

```diff
@@ -76,7 +76,7 @@ server {
       {% endfor %}
     {% endif %}
     {% for limit in location.get('limit_reqs', []) %}
-        limit_req zone={{ limit.zone_name }}{% if limit.get('burst') %} burst={{limit.burst}}{% endif %}{% if limit.get('nodelay') %} nodelay{% endif %};
+        limit_req zone={{ limit.zone_name }} burst={{limit.burst}}{% if limit.get('nodelay') %} nodelay{% endif %};
```

Comment:
i made it required because at the time i thought that in order to perform well it needed the attribute. now i'm not as sure